vendor/docs-page-shell/page-shell-styles.css

```diff
@@ -58,24 +58,6 @@
   color: inherit;
   text-decoration: none;
 }
-@font-face {
```

Comment:
Similar to my comment above, this doesn't have any real effect since Assembly loads these fonts anyway.